cpg_workflows/stages/large_cohort.py

```diff
@@ -12,73 +14,110 @@
     get_workflow,
     stage,
 )
+from metamist.graphql import gql, query
 
-from .genotype import Genotype
+if TYPE_CHECKING:
+    from graphql import DocumentNode
 
+    from hailtop.batch.job import PythonJob
 
-@stage(required_stages=[Genotype])
+
+@stage(analysis_type="combiner", analysis_keys=["combiner"])
 class Combiner(CohortStage):
     def expected_outputs(self, cohort: Cohort) -> Path:
-        if vds_version := get_config()['workflow'].get('vds_version'):
-            vds_version = slugify(vds_version)
-            if not vds_version.startswith('v'):
-                vds_version = f'v{vds_version}'
-
-        vds_version = vds_version or get_workflow().output_version
-        return cohort.analysis_dataset.prefix() / 'vds' / f'{vds_version}.vds'
+        workflow_config = config_retrieve("workflow")
+        combiner_config = config_retrieve("combiner")
+        output_vds_name: str = slugify(
+            f"{workflow_config['cohort']}-{workflow_config['sequencing_type']}-{combiner_config['vds_version']}",
+        )
+        return cohort.analysis_dataset.prefix() / "vds" / f"{output_vds_name}.vds"
+
+    def get_vds_ids_output(self, vds_id: int) -> list[str]:
+        get_vds_analysis_query: DocumentNode = gql(
+            """
+            query getVDSByAnalysisIds($vds_id: Int!) {
+                analyses(id: {eq: $vds_id}) {
+                    output
+                    sequencingGroups {
+                        id
+                    }
+                }
+            }
+        """,
+        )
+        query_results: dict[str, Any] = query(get_vds_analysis_query, variables={"vds_id": vds_id})
+        vds_path: list[str] = [query_results["analyses"][0]["output"]]
+        vds_ids: list[str] = [sg["id"] for sg in query_results["analyses"][0]["sequencingGroups"]]
```

Comment:
```suggestion
        vds_sgids: list[str] = [sg["id"] for sg in query_results["analyses"][0]["sequencingGroups"]]
```

Can you rename to `vds_sgids` to differentiate from `vds_id` above which is a very different ID?

Reply:
something similar is referred to as `vds_sg_ids` below, so would have to avoid a clash there...